app/routes/events/view/index.js

```diff
@@ -1,11 +1,20 @@
 import Route from '@ember/routing/route';
+import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
 
-export default Route.extend({
-  async model() {
+export default class extends Route.extend(EmberTableRouteMixin) {
+
+  async model(params) {
     let eventDetails = this.modelFor('events.view');
+    const searchField = 'name';
+    let filterOptions = [];
+    filterOptions = this.applySearchFilters(filterOptions, params, searchField);
     let sponsorQueryObject = {
-      'page[size]': 10
+      filter         : filterOptions,
+      'page[size]'   : params.per_page || 10,
+      'page[number]' : params.per_page || 1
     };
+    sponsorQueryObject = this.applySortFilters(sponsorQueryObject, params);
+
 
     return {
```

Comment:
what is this, you just have to copy the format from other tables, if you don't understand the underlying logic of how things work.

Reply:
@CosmicCoder96 Everywhere this is the same way `Sorting` is applied on the object, I have just renamed the `queryString` to `sponsorQueryObject` to maintain the convention as of before `Ember Tables`